rlberry/agents/agent.py

```diff
@@ -168,6 +161,7 @@ def thread_shared_data(self):
     @abstractmethod
     def fit(self, budget: int, **kwargs):
         """
+        Abstract method to be override by the 'inherited agent' developer.
```

Comment:
overridden
I don't understand the word "developer" at the end. Remove it ?

Reply:
done !